Description: Delete third_party dependencies
 The reason for this is that there are a few git submodules in this folder,
 which need to be installed as packages in order to build. Cannot be applied
 upstream because Intel still builds with git submodules.
Author: Shane McKee <shane.mckee@canonical.com>
Last-Update: 2024-07-25
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -44,7 +44,6 @@
 
 add_subdirectory(compiler)
 add_subdirectory(firmware)
-add_subdirectory(third_party)
 add_subdirectory(umd)
 add_subdirectory(validation)
 
